vm/src/compile.rs

```diff
@@ -318,8 +351,14 @@ impl Compiler {
                 self.compile_expression(b);
                 self.emit(Instruction::StoreSubscript);
             }
+            ast::Expression::Attribute { value, name } => {
+                self.compile_expression(value);
+                self.emit(Instruction::StoreAttr {
```

Comment:
This should be Instruction::get_attr, not storeAttr right? When we face an attribute expression, we must fetch the attribute.

Reply:
That's inside compile_store.